docs/SRS-Volere/SRS.tex

```diff
@@ -252,22 +252,52 @@ \subsection{Products That Can Be Copied}
 
 \section{New Problems}
 \subsection{Effects on the Current Environment}
-\lips
+\begin{enumerate} [{}1.]
+  \item The application should not create redundant workload for tasks already carried out by existing solutions.
+\end{enumerate}
 \subsection{Effects on the Installed Systems}
-\lips
+\begin{enumerate} [{}1.]
+  \item The application should not affect the system it is being run on other than taking in the required inputs
+and providing the appropriate outputs.
+  \item The application should not affect the systems it is interfacing with other than retrieving
+data, and sending data if necessary.
+\end{enumerate}
 \subsection{Potential User Problems}
-\lips
+\begin{enumerate} [{}1.]
+  \item The user may not have access to the internet.
+  \item The user may not have a device which can run the application.
+\end{enumerate}
 \subsection{Limitations in the Anticipated Implementation Environment That May
 Inhibit the New Product}
-\lips
+N/A
 \subsection{Follow-Up Problems}
-\lips
+\begin{enumerate} [{}1.]
+  \item Business processes might change, changing the requirements of the application.
+  \item New software solutions may be introduced which make some features redundant.
+  \item Regulations may change adding or removing requirements.
+\end{enumerate}
 
 \section{Tasks}
 \subsection{Project Planning}
-\lips
+Project deliverables should be completed by the deadlines given in the course outline. GitHub will be used
+to track project milestones and tasks. Tasks will be assigned to individual team members or to groups. 
+All work will be reviewed by other members of the team before being committed to the project. Feedback
+received from stakeholders, TAs, or the professor will be implemented in the project, and requirements will
```

Comment:
since we're using Latex now we should follow the style guide from the prof https://github.com/Spitgranger/capstone/blob/main/docs/Checklists/Writing-Checklist.pdf, the line lengths here is a little bigger than 80 I think

Reply:
Sorry, I was using a 100c limit instead of 80c, mb.